geopyspark/geotrellis/layer.py

```diff
@@ -1728,6 +1729,39 @@ def slope(self, zfactor_calculator):
 
         return TiledRasterLayer(self.layer_type, srdd)
 
+    def crop(self, area_of_interest, crop_options=None):
```

Comment:
I'm not sure... Why do we have both available in GT? I'd assume the differences in implementation makes one operation more optimal than the other in certain situations.

Reply:
For a single tile case `mask` will turn some pixels into `NODATA` and `crop` will/may reduce the size of the tile, dropping pixels. For RDD case either may drop tiles that don't intersect AOI because having `NODATA` tiles is very costly and both must preserve the square shape of the tile because thats the requirement of `TileLayerRDD`. 

Honestly both got implemented for RDD because mostly we didn't think very hard about it at the time and perhaps to carry over both method names into distributed case. This may be a chance to avoid same mistake here.

Reply:
I see. In that case, it probably would be better to close this PR then; and point to `mask` as the go to crop method.